### PR TITLE
fixing docker image push

### PR DESCRIPTION
**Pull request by dharmendrakariya**
Did you run `make format && make check`?

Fixes #	

Docker build and push to ```skycoin/skywire``` dockerhub

 Changes:	
-	Makefile and appveyor CI

How to test this PR:

- Docker Username and Password need to be set on CI envs

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@
 VERSION := $(shell git describe)
 RFC_3339 := "+%Y-%m-%dT%H:%M:%SZ"
 COMMIT := $(shell git rev-list -1 HEAD)
-BRANCH := latest
+BRANCH := develop
```
Comment (mrpalide) [inline Makefile]: I think it's better to remove this part, and add something like `DOCKER_TAG` to instead.
Also add `DOCKER_TAG=master` and `DOCKER_TAG=develop` on AppVeyor yaml file in deploy jobs.
The docker-build command should change to `./ci_scripts/docker-push.sh -t ${DOCKER_TAG} -b`

**Review by mrpalide — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (dharmendrakariya) [reply Makefile]: I changed the .appveyor.yaml!

**Review by mrpalide — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -213,7 +211,7 @@ github-release:
 
 
 build-docker: ## Build docker image
-	./ci_scripts/docker-push.sh -t ${BRANCH} -b
+	./ci_scripts/docker-push.sh -t latest -b
```
Comment (ersonp) [inline Makefile:214]: This needs to be `prod`

Comment (dharmendrakariya) [reply Makefile:214]: I dont think so! that is in general docker build process! which should be tagged latest.

**Pull request merged**